test/ad_service/query/for_display_test.exs

```diff
@@ -83,6 +83,27 @@ defmodule AdService.Query.ForDisplayTest do
     {:ok, %{audience: audience, creative: creative, campaign: campaign}}
   end
 
+  describe "fallback_ad_by_property_id/1" do
+    test "it returns advertisements for the fallback ad in the audience the property is associated with" do
+      fallback_campaign = insert(:campaign)
+
+      property =
+        insert(:property, audience: insert(:audience, fallback_campaign_id: fallback_campaign.id))
+
+      assert AdService.Query.ForDisplay.fallback_ad_by_property_id(property.id) ==
+               %AdService.Advertisement{
+                 body: "This is a Test Creative",
+                 campaign_id: fallback_campaign.id,
+                 headline: "Creative Headline",
+                 image_url: "http://example.com/some.png",
+                 ecpm: Decimal.new("2.00"),
+                 campaign_name: "Test Campaign",
+                 small_image_object: nil,
```

Comment:
Just curious.. I noticed above you removed the small object. Is this not populated in the test for a reason?